driver-core/src/main/com/mongodb/internal/async/client/AsyncAggregateIterableImpl.java

```diff
@@ -40,7 +40,7 @@
 import static com.mongodb.assertions.Assertions.notNull;
 
 
-class AggregateIterableImpl<TDocument, TResult> extends MongoIterableImpl<TResult> implements AggregateIterable<TResult> {
+class AsyncAggregateIterableImpl<TDocument, TResult> extends AsyncMongoIterableImpl<TResult> implements AsyncAggregateIterable<TResult> {
```

Comment:
I didn't have to change the package-private names but it will help _us_ a bit when we search by class name in IntelliJ

Reply:
Ack